drivers/block/rbd.c

```diff
@@ -217,12 +217,14 @@ struct rbd_obj_request;
 typedef void (*rbd_obj_callback_t)(struct rbd_obj_request *);
 
 enum obj_request_type {
-	OBJ_REQUEST_NODATA, OBJ_REQUEST_BIO, OBJ_REQUEST_PAGES
+	OBJ_REQUEST_NODATA = 1,
```

Comment:
Hi @idryomov could you help to explain more about the background of this change? In my opinion, enum starts from 0 is good for array. for example:
```
enum operation_type {
          TYPE_ZERO = 0,
          TYPE_ONE,
          TYPE_TWO,
          TYPE_MAX,
};

int operation_counts[TYPE_MAX];

int show_operation_count(enum  operation_type) {
      if (operation_type >= TYPE_MAX)
            return -EINVAL;
      return operation_counts[operation_type]; 
}
```
Thanx

Reply:
Well, we don't have obj_req arrays indexed by OBJ_REQUEST_*.  It's used as a type identifier, and starting at 1 makes it easier to spot uninitialized use when prototyping or debugging.

Reply:
okey, makes sense.